apps/frontend/src/libs/components/table/styles.module.css

```diff
@@ -40,7 +40,6 @@
 
 .table .table-body {
 	display: block;
-	max-height: 450px;
```

Comment:
You shouldn't change this. This is limited to make the table scrollable.

Reply:
You're right, will get this back to the orignal version